### PR TITLE
Cd bugs

### PR DESCRIPTION
**Pull request by azimov**
Fixes for issues #164 #161 #160

### CODE REVIEW

File: man/readColDefsFromJSON.Rd
Comment (nhall6) [inline man/readColDefsFromJSON.Rd:1]: we want to remove this completely

File: man/saveColDefsAsJSON.Rd
Comment (nhall6) [inline man/saveColDefsAsJSON.Rd:1]: we want to remove this completely

File: man/readColDefsFromJSON.Rd
Comment (azimov) [reply man/readColDefsFromJSON.Rd:1]: This documentation is useful for developers using these functions so should be included in the package. We just don't want to export the functions

File: man/saveColDefsAsJSON.Rd
Comment (azimov) [reply man/saveColDefsAsJSON.Rd:1]: see above comment

**Pull request merged**